src/manifold/src/smoothing.cpp

```diff
@@ -434,7 +434,7 @@ std::vector<Smoothness> Manifold::Impl::SharpenEdges(
     float minSharpAngle, float minSmoothness) const {
   std::vector<Smoothness> sharpenedEdges;
   const float minRadians = glm::radians(minSharpAngle);
-  for (int e = 0; e < halfedge_.size(); ++e) {
+  for (int e = 0; e < static_cast<int>(halfedge_.size()); ++e) {
```

Comment:
Why not the same as the change just below?

Reply:
Probably some error when I was too sleepy writing this, will fix.